docs_user/modules/proc_adopting-autoscaling.adoc

```diff
@@ -38,31 +37,13 @@ spec:
 '
 ----
 
-.Verification
-
-. If autoscaling services are enabled, inspect Aodh pods:
+. Create aodh alarms with the `PrometheusAlarm` alarm type:
 +
-----
-AODH_POD=`oc get pods -l service=aodh -n openstack | tail -n 1 | cut -f 1 -d' '`
-oc exec -t $AODH_POD -c aodh-api -- cat /etc/aodh/aodh.conf
-----
-
-. Check whether Aodh API service is registered in {identity_service}:
+[NOTE]
+You must use the `PrometheusAlarm` alarm type instead of `GnocchiAggregationByResourcesAlarm`.
 +
```

Comment:
Does it make sense to move the "alarm creation" step right after the step to deploy the autoscaling services? 
And is creating an alarm optional?

Reply:
I think we should move the procedure to create alarm into verification as it makes more sense to execute that command after we verify that Aodh is working fine. Yes, creating alarm is optional.